reference/7.2/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -451,6 +451,18 @@ intro
 -----
 Once upon a time...
 ```
+Given a list of indices, the operator returns a list of members corresponding to those indices.
+
+```powershell
+-JOIN '123'[ 2 .. 0 ] -EQ '321'
+```
```

Comment:
`..` is the range operator so it is better described as: "Give a range of indices...".

The example is overly complicated. It is better to just show the output. For example:

```
PS> '123'[2..0]

3
2
1
```